solutions/aspnet/Makefile

```diff
@@ -17,13 +17,28 @@ run: appdir private.pem
 	test -f server.pid && kill -9 `cat server.pid` || true
 	$(MYST) package appdir private.pem config.json
 	myst/bin/$(APPNAME) $(OPTS) & echo $$! > server.pid
-	sleep 15
+	sleep 60
```

Comment:
This would increase pipeline run time by 45 seconds. On my VM, `sleep 20` is enough to launch the server.

Reply:
My machine needs 50 seconds.